files/plir-hapifhir/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+services:
+  plir-hapi:
+    image: mozzy11/openmrs-plir:basic_auth
```

Comment:
haven't you managed to host this outside your env?

Reply:
not sure ,the easiest procedure to host it on the OpenMRS repo

Reply:
Yeah, we either need a image that it's 'popular' and already available in dockerhub, or we need to host it ourselves. 
Can you start by pointing the repository where you built it from?

Also, isn't this image already available in dockerhub?

Reply:
The guidelines are in the README https://github.com/openmrs/openmrs-contrib-ansible-docker-compose/blob/master/README.md

Reply:
I think you missed healthchecks

Reply:
@cintiadr here is the repository i built it from  https://github.com/ibacher/openmrs-contrib-plm-fhir-server

Sure let me add the health checks

Reply:
Unless you've forked the repo I setup, I don't think it will actually use the MySQL server (i.e., [it's configured to use H2](https://github.com/ibacher/openmrs-contrib-plm-fhir-server/blob/1b62d8bf8116a08c719b33f5f69c6a9abb39d0d9/src/main/resources/application.yaml#L3)).

Reply:
@ibacher your right , the app by default runs on H2